projects/demo/src/modules/services/alerts/examples/2/index.ts

```diff
@@ -38,8 +38,8 @@ export class TuiNotificationsExampleComponent2 {
     }
 
     showDepositAlert(): void {
-        this.notificationsService
-            .show(this.depositTemplate || '', {
+        this.alertService
+            .open(this.depositTemplate || ('' as any), {
```

Comment:
can we add TODO to remove it later?

looks terrible(

Reply:
Of course, I think this is a problem that will need to be solved later

Reply:
Why string does not pass as a valid argument?

Reply:
fixed